test/test_site/index.md

```diff
@@ -65,7 +65,27 @@ head: myCustomHead.md
 <panel header="## Outer nested panel" src="testPanels/NestedPanel.md" expanded>
 </panel>
 
-# panel with src from another Markbind site
+# Nested panel without src
+<panel header="## Outer nested panel without src" expanded>
+
+  ## Panel content of outer nested panel
+  <panel header="## Inner panel header without src" expanded>
+  
+  ## Panel content of inner nested panel
+  </panel>
+</panel>
+
+# Panel with src from another Markbind site
 <panel header="## Panel with src from another Markbind site header" src="sub_site/index.md" expanded>
 </panel>
 </div>
+
+# Unexpanded panel
+<panel header="## Unexpanded panel header">
+
+  ## Panel content of unexpanded panel
```

Comment:
As discussed: append with " - should not appear in search data". Same for line 87 and line 89.